src/solvers/sweep_solve.jl

```diff
@@ -1,40 +1,30 @@
 
-region_callback(problem; kws...) = nothing
-
-function sweep_callback(problem; outputlevel, sweep, nsweeps, kws...)
-  if outputlevel >= 1
-    println("Done with sweep $sweep/$nsweeps")
-  end
+function default_region_callback(sweep_iterator; kwargs...)
+  return sweep_iterator
 end
-
+function default_sweep_callback(sweep_iterator; kwargs...)
+  return sweep_iterator
+end
```

Comment:
Maybe it's better to remove the `kwargs...` to make sure we don't accidentally pass unsupported keyword arguments.